app-web/nginx/spec

```diff
@@ -1,5 +1,9 @@
 # Stable branch
 VER=1.28.0
-SRCS="tbl::https://nginx.org/download/nginx-$VER.tar.gz"
-CHKSUMS="sha256::c6b5c6b086c0df9d3ca3ff5e084c1d0ef909e6038279c71c1c3e985f576ff76a"
+SRCS="tbl::https://nginx.org/download/nginx-$VER.tar.gz \
+      git::commit=a71f9312c2deb28875acc7bacfdd5695a111aa53;rename=ngx_brotli::https://github.com/google/ngx_brotli.git"
+CHKSUMS="sha256::c6b5c6b086c0df9d3ca3ff5e084c1d0ef909e6038279c71c1c3e985f576ff76a \
+         SKIP"
 CHKUPDATE="anitya::id=5413"
+SUBDIR="nginx-$VER"
+REL=1
```

Comment:
Instead of bumping REL, you should be using a compound version, something like:

```
1.28.0+ngx_brotli1.0.0~rc1
```

So that all components of this compound source is represented.